contracts/src/PriceFeeds/MainnetPriceFeedBase.sol

```diff
@@ -70,11 +83,11 @@ abstract contract MainnetPriceFeedBase is IPriceFeed, Ownable {
         return (scaledPrice, oracleIsDown);
     }
 
-    function _disableFeedAndShutDown(address _failedOracleAddr) internal returns (uint256) {
+    function _shutDownAndSwitchToLastGoodPrice(address _failedOracleAddr) internal returns (uint256) {
         // Shut down the branch
         borrowerOperations.shutdownFromOracleFailure(_failedOracleAddr);
```

Comment:
It’s not related to this PR, but I wonder if we could emit the `ShutDownFromOracleFailure` here, so we don’t need to pass `_failedOracleAddr` as param to `BorrowerOperations`.

Reply:
Sure, makes sense

Reply:
This is not done yet, right?